services/player.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+
+const api = require('./api');
+const _ = require('lodash');
+//  Doesn't work without /lib/stripIndents
+const stripIndents = require('common-tags/lib/stripIndents');
+const CURR_SEASON = 8;
+
+module.exports = {
+  getTeamInfo: function (...players) {
+    if (!players || players.length < 2) {
+      return Promise.resolve('Invalid arguments provided. Try !help');
+    }
+    const idPromise = getPlayerIDs(players);
+    const teamPromise = idPromise.then(ids => getTeam(ids));
+    return Promise.all([idPromise, teamPromise]).then(([ids, team]) => {
+      return formatTeamData(team.data.data.find((obj) => {
+        return ids.every(ele => obj.attributes.stats.members.includes(ele));
+      }).attributes);
+    });
+  }
+};
+
+function getPlayerIDs(...players) {
+  const params = { 'filter[playerNames]': players.join() };
+  return api.get('/players', { params })
+    .then(res => res.data.data.map(player => player.id));
+}
+
+function getTeam(...ids) {
+  const params = { 'tag[season]': CURR_SEASON, 'tag[playerIds]': ids.join() };
+  return api.get('/teams', { params });
+}
+
+function formatTeamData(teamData) {
+  return stripIndents`
+    Team Name: ${teamData.name}
+    Rank: ${formatLeague(teamData.stats.league)} League | Division ${teamData.stats.division}
+    Division Rating: ${teamData.stats.divisionRating}
+    Wins: ${teamData.stats.wins}
+    Losses: ${teamData.stats.losses}
+    Win Rate: ${_.round((teamData.stats.wins / (teamData.stats.losses + teamData.stats.wins) * 100), 2)}%
```

Comment:
Few things:
1. avoid calculating stuff directly inside template, calculate it outside and just reference it here
2. because you use `_.round` only import it as:
```js
const round = require('lodash/round');
```